src/components/commons/MetascanLinkButton.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import classNames from "classnames";
+import { useTheme } from "@contexts/ThemeContext";
+
+interface ViewOnMetascanButtonProps {
+  href?: string;
+  customStyle?: string;
+}
+
+export function MetascanLinkButton({
+  href,
+  customStyle,
+}: ViewOnMetascanButtonProps): JSX.Element {
+  return (
+    <a
+      href={href}
+      target="_blank"
+      rel="noreferrer"
+      data-testid="metascan-link-button"
+    >
+      <button
+        type="button"
+        data-testid="view-on-metachain-button"
+        className={classNames(
+          "border-[0.5px] rounded-[30px] border-gray-200 dark:border-gray-700",
+          "flex items-center gap-2",
+          "py-1 px-2 text-sm font-medium text-gray-700 dark:text-gray-300 tracking-[0.0025em]",
```

Comment:
expected padding is 6px, 8px, 6px, 8px

Reply:
<img width="248" alt="image" src="https://github.com/BirthdayResearch/defiscan.live/assets/53080940/552e5860-384a-49c0-979b-2c0319230288">
alignment for ipad screen

Reply:
<img width="1027" alt="image" src="https://github.com/BirthdayResearch/defiscan.live/assets/53080940/93f8ec39-9cf2-4251-8e3b-1e40bf44af18">
Its not fixed